Costume/147_Dratini.yaml

```diff
@@ -0,0 +1,20 @@
+species: Dratini
+imagename: Dreepy
+displayname: Dreepy
+setname: Costume
+tags: [costume]
+item: [Life Orb]
+ability: [Clear Body]
+moves:
+    - [Draco Meteor]
+    - [Bite]
```

Comment:
I would consider sucker punch over bite as draco meteor is the far superior option.